packages/hash/frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card.tsx

```diff
@@ -164,6 +171,33 @@ const DisabledCheckboxCell = ({
   );
 };
 
+const usePropertyTypeVersions = (
+  propertyTypeId: VersionedUri,
+  propertyTypes?: Record<VersionedUri, PropertyType> | null,
+) => {
+  return useMemo(() => {
+    const baseUri = extractBaseUri(propertyTypeId);
+
+    const propertyTypeIds = propertyTypes
+      ? Object.values(propertyTypes).map(({ $id }) => $id)
+      : [];
+
+    const versions = propertyTypeIds
+      .filter((versionedUri) => {
+        return versionedUri.startsWith(`${baseUri}v/`);
+      })
```

Comment:
If you had the `subgraph` here this could be replaced by `getPropertyTypesByBaseUri(subgraph, baseUri)`

Reply:
Using  `getPropertyTypesByBaseUri(subgraph, baseUri)` in 60c5ba7e3bc80cd4a33cae6ec47d2ec8103afb9e